Use the LibreOffice bundled OpenSSL DLL suffix on all Windows architectures

--- python3/PCbuild/openssl.props.orig
+++ python3/PCbuild/openssl.props
@@ -11,8 +11,6 @@
   </ItemDefinitionGroup>
   <PropertyGroup>
     <_DLLSuffix>-3</_DLLSuffix>
-    <_DLLSuffix Condition="$(Platform) == 'ARM'">$(_DLLSuffix)-arm</_DLLSuffix>
-    <_DLLSuffix Condition="$(Platform) == 'ARM64'">$(_DLLSuffix)-arm64</_DLLSuffix>
     <OpenSSLDLLSuffix Condition="$(OpenSSLDLLSuffix) == ''">$(_DLLSuffix)</OpenSSLDLLSuffix>
   </PropertyGroup>
   <ItemGroup>
